maps-ktx/src/main/java/com/google/maps/android/ktx/StreetViewPanoramaFragment.kt

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
That's primarily an organizational preference. The `src/main/java` path is what Android Studio creates for you by default when you create a new project. You can still add `.kt` files in a `java` dir and vice versa. 

In most cases I prefer to intermix `.kt` and `.java` files in the same directory without having to create a `kotlin` path so that you don't have to keep track of consistent package names. However, having separate language-specific directories I think is good practice in the case of samples.